packages/oasis-actions/src/types/ajna/AjnaEarnPosition.ts

```diff
@@ -154,4 +155,30 @@ export class AjnaEarnPosition implements IAjnaEarn {
       this.rewards,
     )
   }
+
+  reopen(quoteTokenAmount: BigNumber, priceIndex: BigNumber) {
+    return new AjnaEarnPosition(
+      this.pool,
+      this.owner,
+      quoteTokenAmount,
+      priceIndex,
+      this.stakedNftId,
+      this.collateralPrice,
+      this.quotePrice,
+      this.rewards,
+    )
+  }
```

Comment:
is reopen needed? Can we just deposit into empty position?

Reply:
yes, I tried with deposit initially and I would need to overwrite princeIndex as well since the one from empty position is 0